client/scss/components/_button.scss

```diff
@@ -321,35 +321,42 @@
 }
 
 .w-header-button {
-  // Temporarily copied from .c-sf-add-button
   display: flex;
   align-items: center;
   justify-content: center;
   gap: theme('spacing.1');
-  height: theme('spacing.slim-header');
+  height: theme('spacing.7');
```

Comment:
Hmm, interesting. I thought it was nice to have the button be a similar height to other ones in the slim header, but I suppose not a must.